Python/bytecodes.c

```diff
@@ -4002,46 +4005,46 @@ dummy_func(
 
         inst(CACHE, (--)) {
             TIER_ONE_ONLY
-            assert(0 && "Executing a cache.");
-            Py_UNREACHABLE();
+            assert(0);
```

Comment:
I kind of liked the original weirdly phrased `assert`, because when you run in debug mode it prints the text of the expression, so you know what's wrong without looking up the source line. (Same below.)

(Agreed that the fatal error is better than unreachable.)

Reply:
I've reinstated it and the one below